CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 -   `Interpolant` now takes `x` and `y` instead of a single `data` entry ([#1312](https://github.com/pybamm-team/PyBaMM/pull/1312))
 -   Boolean model options ('sei porosity change', 'convection') must now be given in string format ('true' or 'false' instead of True or False) ([#1280](https://github.com/pybamm-team/PyBaMM/pull/1280))
 -   Operations such as `1*x` and `0+x` now directly return `x`. This can be bypassed by explicitly creating the binary operators, e.g. `pybamm.Multiplication(1, x)` ([#1252](https://github.com/pybamm-team/PyBaMM/pull/1252))
+-   ```'Cell capacity [A.h]'``` has been renamed to ```'Nominal cell capacity [A.h]'```. ````'Cell capacity [A.h]'``` will be deprecated in the next release. ([#1352](https://github.com/pybamm-team/PyBaMM/pull/1352))
```

Comment:
The last 'Cell capacity [A.h]' has four opening ` but only three closing ones. BTW, in this case you can use only one opening and closing ` (see line above, for example).

Reply:
Done, thanks!